api/admin.go

```diff
@@ -311,22 +336,44 @@ func (a *API) adminUserCreate(w http.ResponseWriter, r *http.Request) error {
 		return internalServerError("Error creating user").WithInternalError(err)
 	}
 
-	if user.AppMetaData == nil {
-		user.AppMetaData = make(map[string]interface{})
+	user.AppMetaData = map[string]interface{}{
+		// default to the first provider in the providers slice
+		"provider":  providers[0],
+		"providers": providers,
 	}
-	user.AppMetaData["provider"] = "email"
-	user.AppMetaData["providers"] = []string{"email"}
 
-	if params.BanDuration != "" {
-		duration, terr := time.ParseDuration(params.BanDuration)
-		if terr != nil {
-			return badRequestError("Invalid format for ban_duration: %v", terr)
+	err = db.Transaction(func(tx *storage.Connection) error {
+		if terr := tx.Create(user); terr != nil {
+			return terr
 		}
-		t := time.Now().Add(duration)
-		user.BannedUntil = &t
-	}
 
-	err = db.Transaction(func(tx *storage.Connection) error {
+		var identities []models.Identity
+		if user.GetEmail() != "" {
+			identity, terr := a.createNewIdentity(tx, user, "email", structs.Map(provider.Claims{
+				Subject: user.ID.String(),
+				Email:   user.GetEmail(),
+			}))
+
+			if terr != nil {
+				return terr
+			}
+			identities = append(identities, *identity)
+		}
+
+		if user.GetPhone() != "" {
+			identity, terr := a.createNewIdentity(tx, user, "phone", structs.Map(provider.Claims{
+				Subject: user.ID.String(),
+				Email:   user.GetPhone(),
```

Comment:
```suggestion
				Phone:   user.GetPhone(),
```

Should this be phone?